### ISSUE TITLE
Unicode failure?

### ISSUE THREAD

**Issue by NMonteix** [stefaneichert/OpenLidarToolbox#12]
Here is what I get trying to use 0NE or Create base data:
File "C:\Users/***/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\LAStools\lastools\core\data_compression\laszip.py", line 66, in processAlgorithm
LastoolsUtils.run_lastools(commands, feedback)
File "C:\Users/***/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\LAStools\lastools\core\utils\utils.py", line 71, in run_lastools
feedback.pushConsoleInfo(output.decode("utf-8"))
^^^^^^^^^^^^^^^^^^^^^^
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8a in position 15: invalid start byte

LAS files (three attempts with three different files) come from here: https://diffusion-lidarhd.ign.fr/

None of the solutions found on the Internet has helped...

**Comment by stefaneichert** [comment]
Hi! Thanks for reporting. Could you please also provide me with the information on:
Operating System, Qgis Version, OpenLidarToolbox verision, Lastools version, Whitebox Tools version and attach one of the files that it did not work with? 
Then I can try to reproduce the error and evaluate the reason.
Best regards,
Stefan

**Comment by NMonteix** [comment]
Hi Stefan,
Thanks for your quick reply.
OS: Win 11 (23H2) | QGis: 3.34.11 | OpenLidar: 2.3 | LastTools: 2.1.1 | Whitebox: 1.0.9
The last file I tried was available here: https://storage.sbg.cloud.ovh.net/v1/AUTH_63234f509d6048bca3c9fd7928720ca1/ppk-lidar/NP/LHD_FXX_0792_6315_PTS_O_LAMB93_IGN69.copc.laz
The message was the same.
All the best
Nicolas

**Comment by stefaneichert** [comment]
Can you please also tell me the CRS of the Laz data?
Thx

**Comment by NMonteix** [comment]
Hi Stefan
I hope you got my answer, even if it seems not (the answer was IGNF:LAMB93).
Anyway, I re-installed everything and it almost worked. At first, I have had the same [issue](https://github.com/stefaneichert/OpenLidarToolbox/issues/5) as F. Attilio, processing with [this file](https://storage.sbg.cloud.ovh.net/v1/AUTH_63234f509d6048bca3c9fd7928720ca1/ppk-lidar/NP/LHD_FXX_0792_6314_PTS_O_LAMB93_IGN69.copc.laz).
But it worked on the second attempt with other files (gorgeous results).
I made a second attempt on the first file I tried the new installation on. The longest processing ever, but at last it worked.
During all processes, I had the same error message ("ERROR 6: z5hz3jp.tif, band 1: SetColorTable() only supported for Byte or UInt16 bands in TIFF format.")
Thanks and all the best
Nicolas
